lib/web_ui/lib/src/engine/html/shaders/image_shader.dart

```diff
@@ -166,7 +166,7 @@ class EngineImageShader implements ui.ImageShader {
     vertices[10] = vRect.left;
     vertices[11] = vRect.top;
 
-    Object? positionAttributeLocation =
+    final Object positionAttributeLocation =
```

Comment:
Not intended by me, but when I changed to `final` the analyzer realized that the value is initialized to a non-null value and because it's `final` it stays non-null forever. So making the type nullable no longer makes sense.